src/lib/requireStandingData.ts

```diff
@@ -9,11 +10,24 @@ const versions = Object.keys(packageInfo.dependencies)
   .filter((d) => d.startsWith("bichard7-next-data-"))
   .reduce((acc: KeyValue<StandingData>, v) => {
     const version = v.replace("bichard7-next-data-", "")
-    acc[version] = require(v)
+    acc[version] = require(`node_modules/bichard7-next-data-${version}/dist`)
```

Comment:
Webpack finds and includes modules names that start with `node_modules/bichard7-next-data-`